src/modules/settings/components/Table.jsx

```diff
@@ -69,23 +69,48 @@ function CustomWhisper(props) {
   );
 }
 
-function useOutsideAlerter(ref, onClick, rowData) {
+function EditCell({rowData, dataKey, onChange, onMouseOver, onMouseLeave, onClick, onPaste, ...props}) {
+  const wrapperRef = useRef(null);
+
   useEffect(() => {
     function handleClickOutside(event) {
-      if (ref.current && !ref.current.contains(event.target)) {
+      if (wrapperRef.current && !wrapperRef.current.contains(event.target)) {
         onClick(rowData.id);
         document.removeEventListener('mousedown', handleClickOutside);
```

Comment:
do we need to clean this up in the unmount?

Reply:
when the `onClick` callback is called it changes status anyway and will unmount